python/sparkdl/graph/spark_utils.py

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
Why is this under graph/ and the keras one under udf/ ? Is this one not meant to be exposed?